jetpack-io/devbox
# Added `devbox generate devcontainer` command

## Summary
- Added `devbox generate devcontainer` command that generates the following files:
  -  `.devcontainer/Dockerfile` and a `.devcontainer/devcontainer.json`
- Added `devbox generate dockerfile` command that generates Dockerfile in same directory as `devbox.json`


## How was it tested?
- compile
- run `devbox generate devcontainer`
- run `ls -la` to confirm the directory (`.devcontainer`) and the files mentioned above are created.
- run `devbox generate dockerfile`
- run `ls -la` to confirm dockerfile is created.
